lisa/node.py

```diff
@@ -443,6 +443,23 @@ def get_working_path(self) -> PurePath:
 
         return self.get_pure_path(result.stdout)
 
+    def find_partition_with_freespace(self, size_in_gb: int) -> str:
```

Comment:
Did you remove it from the old place?

Reply:
Ah this is a bug. Artifact of a rebase, will fix